exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -191,6 +191,14 @@ impl JavaServiceRuntime {
         // Append extra user arguments
         args_builder = Self::add_user_arguments(args_builder, args_append);
 
+        // Log JVM arguments
+        let mut jvm_args_line = String::new();
```

Comment:
I'd suggest to keep this method at the same level of abstraction — i.e., extract this code in a function `log_jvm_arguments(args_builder)` (the steps above have already hidden their implementation details in separate functions).

Reply:
Done